### ISSUE TITLE
Apply CSS to print view

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#1432]
Need fonts at least though really we should have all the CSS apply so it makes sense. Materialize, tweaks, and fonts especially.

**Comment by btopro** [comment]
same as core is now applied. If this needs revisited just open a new issue
